extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -378,6 +382,24 @@ export class AmpStoryPage extends AMP.BaseElement {
     }
   }
 
+  /** @private */
+  maybeSetStoryNextUp_() {
+    const autoAdvanceAttr = this.element.getAttribute('auto-advance-after');
+    const storyNextUpParam = Services.viewerForDoc(this.element).getParam(
+      'storyNextUp'
+    );
+    if (
+      this.isLastPage() &&
+      autoAdvanceAttr === null &&
+      storyNextUpParam !== null
```

Comment:
Nit: please move the `storyNextUpParam` as the first condition so we save one `querySelector` when possible

Reply:
Sounds good, how does this look?